.github/workflows/build-check-install.yaml

```
@@ -176,6 +194,22 @@ jobs:
               writeLines(readLines(check_log))
               stop("⚠ R CMD check has warnings")
             }
+            if ("${{ inputs.check-note-blocklist }}" == "true") {
+              print("Checking notes...")
+              regexes <- "${{ inputs.note-blocklist }}"
+              regexes <- unlist(strsplit(regexes, split = "\n"))
+              lines = paste(readLines(check_log), collapse = " ")
+              notes_result <- lapply(regexes, function(r){
+                if(grepl(r, lines)){
+                  print(r)
+                  return(TRUE)
+                }
+                return(FALSE)
+              })
```

let's convert list into a vector - this is easier to work with
```suggestion
              notes_result <- vapply(
                regexes, 
                function(r){
                  if (grepl(r, lines)) {
                    print(r)
                    return(TRUE)
                  }
                  return(FALSE)
                },
                logical(1)
              )
```